README.md

```diff
@@ -69,6 +70,8 @@ For incremental builds, the `rush build` command can be used to only build packa
 11. Add and commit the changelog JSON files and any API signature updates.
 12. Publish changes on the branch and open a pull request.
 
+> If executing scripts from 'package.json` files in any of the subdirectories, we recommend using pnpm over npm, as it is more performant.
```

Comment:
let's recommend using `rushx` instead since the default package manager is already `pnpm`

Reply:
done